README.md

```diff
@@ -18,8 +18,7 @@ Kafka Manager images come in two flavors:
 docker run -d \
      -p 9000:9000  \
      -e ZK_HOSTS="localhost:2181" \
-     hlebalbau/kafka-manager:stable \
-     -Dpidfile.path=/dev/null
```

Comment:
Is problem with pidfile fixed in newer versions of cmak?

Reply:
I have no idea about `pidfile problem`, but this JVM system property is already passed in `Dockerfile` ENTRYPOINT.

I assume there's no need to duplicate it when running a container.

Correct me if I'm wrong.

Reply:
gotcha, you are right! resolved